### PR TITLE
improved data ETL

### PR DESCRIPTION
**Pull request by galileukim**
Dear Cliar technical team,

Please review the improvements in the data pipeline proposed. There are two major changes, along with a minor one:

1. Major updates: (a) transformation of ETL from a master.R to an index.Rmd, meaning that whenever we run the pipeline, we produce a new bookdown file (more details on our wiki) and (b) improvement in the folder structure, making explicit the order of steps in our pipeline, along with a dedicated folder for our quantitative review (`data-quality-control`), (c) creation of an renv.lock to keep track of all used packages for the `code` pipeline.
2. Minor update: (a) fix projection of maps, removing the `rmapshaper` since the package will be obsolete in October 2023.

Please review if the Shiny App runs as intended in your local machine, after deploying our modified data pipeline. If it does, we are good to go!

### CODE REVIEW

File: .gitignore
Comment (kannanv93) [inline .gitignore:1]: Modifying gitignore file is impacting the deployment. We need to check this before approving this change.

**Review by kannanv93 — COMMENTED** [review]
Need review of .gitignore file before merging with main branch

**Comment by galileukim** [comment]
@kannanv93 that's good to know. What in particular is affecting our deployment? On my end, I have modified the .gitignore with the following. I could revert those changes if needed.

#######################
# renv file
!renv.lock
!settings.json
!activate.R
!.Rprofile.R

**Review by kannanv93 — APPROVED** [review]
Approved

**Comment by galileukim** [comment]
@kannanv93 just wanted to double check: did you run the entire data pipeline using bookdown? and did the app work as intended once you deployed?

**Pull request merged**